keps/sig-node/3695-pod-resources-for-dra/README.md

```diff
@@ -200,8 +208,8 @@ message CDIDevice {
 
 // GetPodResourcesRequest contains information about the pod
 message GetPodResourcesRequest {
-    string name = 1;
-    string namespace = 2;
+    string pod_name = 1;
```

Comment:
This would be a major breaking change.

Reply:
This is already merged code, I am just updating the KEP to the existing code base.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto#L113

Just found that out also.